app/src/utils/madlib/MadLibs.ts

```diff
@@ -37,6 +37,22 @@ function getMadLibPhraseText(madLib: MadLib): string {
   return madLibText;
 }
 
+// Updates a MadLib with a new value in the given phrase segment index
+export function getMadLibWithUpdatedValue(
```

Comment:
Change comment to /** */ style, and change from "Updates a" to "Returns a copy of the"

Reply:
oh good catch